packages/beacon-node/src/chain/blocks/verifyBlock.ts

```diff
@@ -174,6 +174,17 @@ export async function verifyBlocksInEpoch(
           default:
         }
       }
+
+      if (isForkPostFulu(fromForkBoundary.fork)) {
+        const fromBlobParameters = this.config.getBlobParameters(fromForkBoundary.epoch);
+        const toBlobParameters = this.config.getBlobParameters(toForkBoundary.epoch);
+
+        if (toBlobParameters.maxBlobsPerBlock !== fromBlobParameters.maxBlobsPerBlock) {
```

Comment:
```suggestion
        if (toBlobParameters.epoch !== fromBlobParameters.epoch) {
```
I think you should compare `BlobParameters.epoch` here. Since blob schedule only enforces `EPOCH` to be unique, but not `MAX_BLOBS_PER_BLOCK`